### ISSUE TITLE
AllOf is not rendering

### ISSUE THREAD

**Issue by approxit** [asyncapi/docgen#21]
Merging parts of different objects is fun. OpenAPI understands it but unfortunately cant render it properly in editor and offline documentation.

Simplest example:
```
asyncapi: 1.0.0
info:
  title: Example API
  version: "1.0"
topics:
  example.topic:
    publish:
      payload:
        allOf:
          - type: object
            properties:
              foo:
                type: string
          - type: object
            properties:
              bar:
                type: string
```

Results in empty payload table:
![results](https://user-images.githubusercontent.com/6125496/38173119-dafe146c-35b8-11e8-9a25-dcfafb30b06b.png)

AsyncAPI does not mention about its limitations and OpenAPI plays with XOf nicely.

**Comment by fmvilas** [comment]
Yes, it's a known limitation. I should probably add it to the documentation meanwhile it's not supported. PRs are welcome too 😊

**Comment by fmvilas** [comment]
@approxit a solution has been released and pushed to npm under version 1.8.0.

**Comment by approxit** [comment]
Marvelous! Thanks for that.
